Makefile.in

```diff
@@ -83,31 +95,43 @@ reaper-install: reaper
 	install -Dm 755 $(OBJDIR)/$</$< -t $(DESTDIR)$(DATADIR)/$(INSTALLDIR)
 
 
-.PHONY: $(OBJDIR)
 $(OBJDIR):
-	@mkdir -p $(OBJDIR)
+	@mkdir -p $(@)
 
 
 .PHONY: clean
 clean:
-	$(info Cleaning "$(OBJDIR)" build directory)
-	@rm -rf -v $(OBJDIR)
+	$(info :: Cleaning source directory )
+	@rm -rf -v $(OBJDIR) ULWGL/ULWGL_VERSION.json ./$(RELEASEDIR) $(RELEASEDIR).tar.gz
+
+
+RELEASEDIR := $(PROJECT)-$(shell git describe --abbrev=0)
+$(RELEASEDIR):
+	mkdir -p $(@)
+
+.PHONY: release
+release: $(RELEASEDIR) | version
+	$(info :: Creating source distribution for release )
+	mkdir -p $(<)
+	rm -rf ULWGL/__pycache__
+	cp -r ULWGL flatpak subprojects Makefile.in configure.sh README.md LICENSE $(<)
+	tar -cvzf $(<).tar.gz $(<)
```

Comment:
The test files are included in the archive and I'd really not want them them to be distributed...

Other than that, I think everything else is fine and I'll merge this ASAP after you make that change

Reply:
Actually, these are just the source code files and not that actually release archive to be distributed, right?